blog/_posts/2018-07-09-dvteam-activated.md

```diff
@@ -0,0 +1,76 @@
+---
+title: DV Team Activated!
+layout: post
+created_at: Mon Jul 09 2018
+permalink: blog/2018-07-09-dvteam-activated
+current: blog
+author: Violeta
+twitter: artnerdnet
+categories: student-posts
+---
+## Starting a new adventure with RgSoC and Living Style Guide from coding newbies to professional front-end developers in 11 months.
+![Starting the summer](http://artnerd.net/images/7452ee6d-ba5c-41b9-8590-dc45c930b256.jpg)

```

Comment:
@artnerdnet Now that you have added pics in this file, you need to add pic credits as well. Please add them for both pictures in the file.

Reply:
is this needed? it's pictures we took together

Reply:
@artnerdnet Yes, I understand it but adding credits is important because bunch of reasons including legal liability. Please check other blog posts for an example on how to add it [basically it's a one line code using image-credits class].